<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorAnomalyAndMotionDetectionWizard</class>
 <widget class="QWidget" name="ColorAnomalyAndMotionDetectionWizard">
  <layout class="QVBoxLayout" name="verticalLayout_root">
   <property name="leftMargin"><number>0</number></property>
   <property name="topMargin"><number>5</number></property>
   <property name="rightMargin"><number>0</number></property>
   <property name="bottomMargin"><number>0</number></property>
   <item>
    <widget class="QGroupBox" name="groupBox_color">
     <property name="title"><string>Color Anomaly Detection</string></property>
     <layout class="QVBoxLayout" name="verticalLayout_color">
      <item>
       <widget class="QCheckBox" name="enableColorCheckBox">
        <property name="text"><string>Enable Color Anomaly Detection</string></property>
        <property name="checked"><bool>true</bool></property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAggressiveness">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text"><string>How aggressively should ADIAT be searching for anomalies?</string></property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelNote">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text"><string>Note: A higher setting will find more potential anomalies but may also increase false positives.</string></property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="aggressivenessSliderPlaceholder" native="true">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_motion">
     <property name="title"><string>Motion Detection</string></property>
     <layout class="QVBoxLayout" name="verticalLayout_motion">
      <item>
       <widget class="QLabel" name="labelMotionQuestion">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text"><string>Do you want to enable motion detection?</string></property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetRadioButtons" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_radio">
         <item>
          <widget class="QRadioButton" name="radioMotionNo">
           <property name="text"><string>No</string></property>
           <property name="checked"><bool>true</bool></property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioMotionYes">
           <property name="text"><string>Yes</string></property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_motion">
           <property name="orientation"><enum>Qt::Horizontal</enum></property>
           <property name="sizeType"><enum>QSizePolicy::Expanding</enum></property>
           <property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation"><enum>Qt::Vertical</enum></property>
     <property name="sizeType"><enum>QSizePolicy::Expanding</enum></property>
     <property name="sizeHint" stdset="0"><size><width>20</width><height>40</height></size></property>
    </spacer>
   </item>
  </layout>
 </widget>
</ui>

